tasks/sysctl-setup.yml

```diff
@@ -7,7 +7,24 @@
     ansible_distribution != 'Debian'
     or ansible_distribution_major_version | int < 10
 
-# See: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
+# See: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#forwarding-ipv4-and-letting-iptables-see-bridged-traffic
+- name: Ensure relevant kernel module is enabled
+  modprobe:
+    name: br_netfilter
+    state: present
+  loop:
+    - br_netfilter
+    - overlay
+
+- name: Persist kernel modules after reboot
+  lineinfile:
+    path: /etc/sysctl.d/k8s.conf
+    line: "{{ item }}"
+  loop:
+    - "net.bridge.bridge-nf-call-iptables = 1"
+    - "net.bridge.bridge-nf-call-ip6tables = 1"
```

Comment:
We enable bridging for both IPv4 and IPv6 though I realise in some situations this may not be necessary. Perhaps there is a more relevant flag I can use to determine if the user would like to use these lines?